Makefile

```
@@ -22,13 +22,15 @@ default: all
 
 .PHONY: all external-table fdw cli server install install-server stage tar rpm rpm-tar deb deb-tar clean test it help
 
-all: external-table cli server
+all: external-table cli server fdw
 
 external-table:
 	make -C external-table
 
 fdw:
+ifneq ($(FDW_SUPPORT),)
 	make -C fdw
+endif
```

This is conditional on `FDW_SUPPORT` being set to some non-empty value; if it is empty, we silently skip this step. I think it would be helpful if we included a message that we are skipping `fdw` and why

```suggestion
ifneq ($(FDW_SUPPORT),)
	make -C fdw
else
	@echo "The variable FDW_SUPPORT was empty; skipping building the fdw/ directory"
endif
```